aikau/src/main/resources/alfresco/lists/views/layouts/Carousel.js

```diff
@@ -223,13 +224,7 @@ define(["dojo/_base/declare",
             }
             else
             {
-               // Calculate a suitable height...
-               // We're going to set the height to be as big as the viewing port allows
-               // from the starting vertical position of the node
-               var position = domGeom.position(this.domNode);
-               var viewPort = win.getBox();
-               this.height = viewPort.h - position.y;
-               this.height += "px";
+               this.height = this.calculateHeight(this.domNode) + "px";
```

Comment:
I'm not sure it is going to be that common a use case. It would probably be recommended to just call "setHeight", however I added the "calculateHeight" function to address the way this widget was originally written as I didn't want to change it too much initially.